src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -232,17 +232,16 @@ def rename_volume(mgr, volname: str, newvolname: str) -> Tuple[int, str, str]:
 
     return r, outb, ""
 
-def list_volumes(mgr):
+def get_all_volnames(mgr):
```

Comment:
Usually we follow getters and setters pattern when we expect a method to return some attribute. Besides, `list` is inaccurate since we are not listing but only returning volume names. Listing these names is done in volume.py. Finally, just saying `volumes` is ambiguous since we have methods like `open_volume`. Saying `volnames` instead is more precise.